layouts/partials/page-tags.html

```diff
@@ -0,0 +1,7 @@
+<div class="page-tags-flex">
+    {{ range $tag := .Params.Tag }}
```

Comment:
```suggestion
    {{ range $tag := .Params.tags }}
```

After making this change, the remaining pages where `tag:` is used instead of `tags:` should be changed. It is never a good thing when a content template is wrong, because the bad practice result is spread farther along! 😆 😅 

```sh
content/features/_template.en.adoc:6:tag: [""]
content/features/admonitions/_index.en.adoc:5:tag: ["content"]
content/features/author-byline/_index.en.adoc:5:tag: ["content"]
content/features/brand-identity/_index.en.adoc:5:tag: ["design"]
content/features/code-highlighting/_index.en.adoc:5:tag: ["content"]
content/features/redirect/_index.en.adoc:5:tag: ["layouts"]
content/features/tab-tables/_index.en.adoc:5:tag: ["content"]
content/contributing/stakeholders.en.adoc:5:tag: ["unicef"]
content/about/lorem-ipsum.en.adoc:5:tag: ["content", "developers"]
```

Reply:
Fixed in commit 2251ef08f70687c3cd5c6ca5683b0f55cf750f7d.